tsMuxer/h264StreamReader.h

```diff
@@ -3,6 +3,7 @@
 
 #include <map>
 #include <iostream>
+#include <algorithm>

```

Comment:
There is no code in the header which directly uses stuff from `<algorithm>` - why was this moved here from the source file?

Reply:
@xavery This is correct. Reference removed.